python/attributes_maprojid_parkid.py

```diff
@@ -30,11 +33,13 @@ def parkid_parse(x):
 # park_id cleaning
 parkproj_cleaned = pd.DataFrame()
 for i in range(len(parkproj)):
-    parkproj_cleaned = parkproj_cleaned.append(
-        parkid_parse(parkproj.iloc[i, :]))
-
+    parkproj_cleaned = pd.concat([parkproj_cleaned, parkid_parse(parkproj.iloc[i, :])])
```

Comment:
In SQL concat and append are different.  In append if a value is NULL a values is still returned (i.e. XYZ + NULL = XYZ and XYZ + ABC = XYZABC), whereas for concat if a value is NULL no values is returned (i.e. XYZ + NULL = NULL and XYZ + ABC = XYZABC).  Refreshing my memory on pandas and looks like this is not the case for Pandas if the axis is `0`, see [here](https://medium.com/analytics-vidhya/a-tip-a-day-python-tip-5-pandas-concat-append-dev-skrol-18e4950cc8cc#:~:text=Concat%20function%20will%20do%20a,of%20rows%20in%20second%20dataframe.), but I want to double check because I do not want to change the logic.

Reply:
This was a hard change to test as the `parkid_parse` function applies a change to records that that a comma in the `fms_id` field, but no records in the data on my local machine have that comma. So the `parkproj` and `parkproj_cleaned` are exactly the same on my local build. I think this would work if some records had the parsing logic applied based on my understanding of how the function works

Reply:
From my understanding (and from a warning message in the dataloading step), append is supposed to be deprecated in a later version of pandas and they suggest using pandas.concat as a replacement

Reply:
Resolved - discussed last week and confirmed - use concat